src/acceptance/config/config_test.go

```diff
@@ -0,0 +1,241 @@
+package config_test
```

Comment:
There were no tests for `src/acceptance/config/config.go` so I introduced some 👀 , mainly to test https://github.com/cloudfoundry/app-autoscaler-release/pull/2790/files#diff-0663a9f1a7550a6674d39ec1ccba147f1e65544f58e1f5b4b3dec3306b9bf0e3R167-R192